zk.el

```diff
@@ -172,7 +172,7 @@ Must take a single STRING argument."
 See `zk-current-notes' for details."
   :type 'function)
 
-(defcustom zk-format-function nil
+(defcustom zk-format-function #'zk--format-default
   "User-defined function for formatting zk file information.
```

Comment:
Since there's a default, it can just be "Function for formatting...", no need for "User-defined".